examples/reflink_block.rs

```diff
@@ -21,9 +21,16 @@ fn main() -> std::io::Result<()> {
     to_file.set_len(len)?;
 
     let mut offset = 0u64;
-    while offset < len as u64 {
-        println!("reflink {offset}, {cluster_size}");
-        reflink_copy::ReflinkBlockBuilder::new(&from_file, &to_file, cluster_size)
+    while offset < len {
```

Comment:
```suggestion
    while let Some(bytes_left) = NonZeroU64::new(len - offset) {
```

Then the `src_length` below could use this instead of having to `unwrap`

Reply:
with current implementation on Windows, offset could become larger than the file size, causing this code to panic.
it would be easier to keep condition as it is for now